FindRange.java

```diff
@@ -1,16 +1,48 @@
-/*
- * File: FindRange.java
- * Name: 
- * Section Leader: 
- * --------------------
- * This file is the starter file for the FindRange problem.
- */
-
-import acm.program.*;
-
-public class FindRange extends ConsoleProgram {
-	public void run() {
-		/* You fill this in */
-	}
-}
-
+/*
+ * File: FindRange.java

```

Comment:
Your program doesn't work correctly because you have an incorrect value of the SENTINEL. You set it to 20 instead of 0. Also it misses the following two edge cases:

```
If the user enters only one value before the sentinel, the program should report
that value as both the largest and smallest
```
and
```
If the user enters the sentinel on the very first input line, then no values have been
entered, and your program should display a message to that effect.
```